src/site/content/en/blog/variable-fonts/index.md

```diff
@@ -0,0 +1,365 @@
+---
+title: Introduction to variable fonts on the web
+subhead: A new font specification that can significantly reduce font file sizes
+description: >
+  How variable fonts work, how typographers implement variable fonts, and
+  how to work with variable fonts in CSS.
```

Comment:
Updated the subhead and added a description to make the value proposition of variable fonts (VFs) clearer

Reply:
works for me